syncd/Syncd.cpp

```diff
@@ -908,12 +912,12 @@ sai_status_t Syncd::processBulkQuadEventInInitViewMode(
 {
     SWSS_LOG_ENTER();
 
+    const auto objectCount = static_cast<uint32_t>(objectIds.size());
+
     std::vector<sai_status_t> statuses(objectIds.size());
 
-    for (auto &a: statuses)
-    {
-        a = SAI_STATUS_SUCCESS;
-    }
```

Comment:
why this is removeD ?

Reply:
status is not SUCCESS for get operation

Reply:
this was added here becasue of init view mode since operatiom must be success for craete/remove/set

Reply:
Right, so the new code looks like this:

```
const sai_status_t initialObjectStatus = api != SAI_COMMON_API_BULK_GET ? SAI_STATUS_SUCCESS : SAI_STATUS_NOT_EXECUTED;
statuses.assign(statuses.size(), initialObjectStatus);
```

where it is STATUS_NOT_EXECUTED for get and STATUS_SUCCESS for create/set/remove